cron/config/config.yaml

```diff
@@ -22,6 +22,10 @@ completion-threshold: 0.99
 shard-size: 10
 webhook-url: 
 cii-data-bucket-url: gs://ossf-scorecard-cii-data
+# TODO: Temporarily remove SAST and CI-Tests which require lot of GitHub API tokens.
```

Comment:
note: ISE wants to use CI-Tests it in their dashboard

Reply:
To be clear, `CI-Tests` have been disabled for a quite a while now. This PR is not disabling any new checks. 

If we want to re-enable `CI-Tests`, need to look into making the check more performant before adding in back in. Don't have the cycles to do that this quarter. Can you create an issue and maybe we can attempt this in Q1?

Reply:
Good to know! Which other checks are currently disabled?

Reply:
SAST, CI-Test and Contributors. We had even updated the doc with this info, IIRC. Can't seem to find it now.